qa/runtest.sh

```diff
@@ -95,11 +95,12 @@ docker run -d \
 S3_CONTAINER_IP=$(docker inspect --format "{{.NetworkSettings.Networks.${DOCKER_NETWORK}.IPAddress}}" ${TESTS3_CONTAINER_NAME})
 S3_ENDPOINT_URL="http://${S3_CONTAINER_IP}:33000"
 
+mkdir shared
+
 echo "Starting test suite"
-docker run -t \
-  -v "${here}:/f8a_worker:ro,Z" \
+docker run -v "$PWD/shared:/tmp/shared:rw,Z" -t \
```

Comment:
Could you just mount the $PWD, so that you can avoid dealing with extra dir?

Reply:
okay

Reply:
@sawood14012 , This won't work when REBUILD is set to 0. `  -v "${here}:/f8a_worker:ro,Z"` is a must, my argument is that should be sufficient if it is mounted in rw. Let me check from my end to understand the issue which you are facing.

Reply:
okay

Reply:
You can use the following changes. 
https://github.com/arajkumar/fabric8-analytics-worker/commit/d9d3abbddca828d8f2bb44d1fe4765a8087c6431

You wouldn't have seen coverage.xml which is written in root dir because it is in .gitignore.